src/psmoveservice/PSMoveTracker/PSEye/PSEyeVideoCapture.cpp

```diff
@@ -391,13 +394,13 @@ class PSEYECaptureCAM_PS3EYE : public cv::IVideoCapture
     {
         // Enumerate libusb devices
         std::vector<ps3eye::PS3EYECam::PS3EYERef> devices = ps3eye::PS3EYECam::getDevices();
-        std::cout << "ps3eye::PS3EYECam::getDevices() found " << devices.size() << " devices." << std::endl;
+		std::cout << "ps3eye::PS3EYECam::getDevices() found " << devices.size() << " devices." << std::endl;
         
         if (devices.size() > (unsigned int)_index) {
             
             eye = devices[_index];
-            
-            if (eye && eye->init(640, 480, 60, ps3eye::PS3EYECam::EOutputFormat::Bayer))
+
+            if (eye && eye->init(640, 480, 15, ps3eye::PS3EYECam::EOutputFormat::Bayer))
```

Comment:
Do we actually want to make the default 15 FPS? Generally I prefer not to change defaults unless the default is broken.